.github/workflows/pull-request-management.yml

```diff
@@ -123,7 +123,7 @@ jobs:
       fail-fast: true
       matrix:
         avd_scenario: ['build_schemas_and_docs']
-        ansible_version: ['ansible-core>=2.11.3,<2.13.0']
+        ansible_version: ['ansible-core>=2.11.3,<2.15.0,!=2.12.0,!=2.12.1,!=2.12.2,!=2.12.3,!=2.12.4,!=2.12.5,!=2.13.0']
```

Comment:
@ClausHolbechArista  so the title of the PR states `>=2.12.6, <2.15, !=2.13.0`
and we have here
`>=2.11.3,<2.15.0,!=2.12.0,!=2.12.1,!=2.12.2,!=2.12.3,!=2.12.4,!=2.12.5,!=2.13.0'`

is it equivalent?

apologies if I missed that in a previous PR..

Reply:
This cherry pick allows more versions than the original. Hence the discrepency. Let's update the title to the actual changes - `Bump: Support additional ansible-core versions 2.13 and 2.14`

Reply:
done

Reply:
thank you!